app/api/answers/get_answer.go

```diff
@@ -10,38 +10,46 @@ import (
 
 // swagger:operation GET /answers/{answer_id} answers answerGet
 //
-//	---
-//	summary: Get an answer
-//	description: >
-//		Returns the answer identified by the given `{answer_id}`.
+//		---
+//		summary: Get an answer
+//		description: >
+//			Returns the answer identified by the given `{answer_id}`.
 //
-//		- If the user is a participant
-//			- (s)he should have at least 'content' access rights to the `answers.item_id` and
-//			- be a member of the `answers.participant_id` team or
-//				`answers.participant_id` should be equal to the user's self group.
+//			- If the user is a participant
+//				- (s)he (or one of his/her teams) should have at least 'content' access rights to the `answers.item_id` and
+//				- be a member of the `answers.participant_id` team or
+//				  `answers.participant_id` should be equal to the user's self group.
 //
-//		- If the user is an observer
-//			- (s)he should have `can_watch` >= 'answer' permission on the `answers.item_id` and
-//			- be a manager with `can_watch_members` of an ancestor of `answers.participant_id` group.
+//			- If the user is an observer/helper
+//				- (s)he should have `can_watch` >= 'answer' permission on the `answers.item_id` OR
+//				- `can_watch` >= 'result' permission on the `answers.item_id` together with a validated result
+//	        (personally or as a team) on the `answers.item_id`.
 //
-//		If any of the preconditions fails, the 'forbidden' error is returned.
-//	parameters:
-//		- name: answer_id
-//			in: path
-//			type: integer
-//			required: true
-//			format: int64
-//	responses:
-//		"200":
-//			"$ref": "#/responses/itemAnswerGetResponse"
-//		"400":
-//			"$ref": "#/responses/badRequestResponse"
-//		"401":
-//			"$ref": "#/responses/unauthorizedResponse"
-//		"403":
-//			"$ref": "#/responses/forbiddenResponse"
-//		"500":
-//			"$ref": "#/responses/internalErrorResponse"
+//				The user is considered as an observer if (s)he is a manager with `can_watch_members`
+//		 		of an ancestor of `answers.participant_id` group.
+//
+//			  The user is considered as a helper if (s)he is an ancestor of the thread's `helper_group_id`
```

Comment:
ancestor? a descendant!

Reply:
The user is a manager of an ancestor of `answers.participant_id`. So he manages `answers.participant_id` as well. Sounds correct, doesn't it?

Reply:
From the doc "be part of the group the participant has requested help to".

`helper_group_id` is the group of person who can help... so the current user must be part of it.. so descendant

Reply:
fixed